src/views/EditMultisigWallet.vue

```diff
@@ -0,0 +1,752 @@
+                    })
+                }
+            }
+        } else {
+            await this.issueMultisigTx()
+            await this.getAliasInfos()
+            console.log('issueMultisigTx')
+            console.log('pendingSendMultisigTX', this.pendingSendMultisigTX)
+        }
+    }
+
+    async cancelEditMsig() {
+        await this.getAliasInfos()
+        this.mode = 'VIEW'
+    }
+
+    async abortEditMsig() {
+        try {
+            const wallet = this.activeWallet as MultisigWallet
+            if (this.pendingSendMultisigTX) {
+                await wallet.cancelExternal(this.pendingSendMultisigTX?.tx)
+                await this.$store.dispatch('Signavault/updateTransaction')
+                this.getAliasInfos()
+                this.mode = 'VIEW'
+            }
+        } catch (err) {
+            console.error(err)
+        }
+    }
+
+    async signEditMsig() {
+        await this.saveEditMsig()
+        this.mode = 'VIEW'
+    }
+
+    async signMultisigTx() {
+        // @ts-ignore
+        const helpers = this.globalHelper()
+        const wallet = this.activeWallet
+        if (!wallet || !(wallet instanceof MultisigWallet))
+            return console.debug('MultiSigTx::sign: Invalid wallet')
+        if (!this.pendingSendMultisigTX) return console.debug('MultiSigTx::sign: Invalid Tx')
+        try {
+            await wallet.addSignatures(this.pendingSendMultisigTX?.tx)
+            helpers.dispatchNotification({
+                message: this.$t('notifications.multisig_transaction_saved'),
+                type: 'success',
+            })
+            this.$store.dispatch('Signavault/updateTransaction')
+        } catch (e: any) {
+            helpers.dispatchNotification({
+                message: this.$t('multisig_transaction_not_saved'),
+                type: 'error',
+            })
+        }
+    }
+
+    async issueMultisigTx() {
+        // @ts-ignore
+        let { dispatchNotification } = this.globalHelper()
+        const wallet = this.activeWallet
+        if (!wallet || !(wallet instanceof MultisigWallet))
+            return console.error('MultiSigTx::sign: Invalid wallet')
+        if (!this.pendingSendMultisigTX) return console.error('MultiSigTx::sign: Invalid Tx')
+        try {
+            await this.updateMultisigTxDetails()
+            await wallet.issueExternal(this.pendingSendMultisigTX?.tx)
+            this.$store.dispatch('Signavault/updateTransaction')
+            dispatchNotification({
+                message: this.$t('notifications.execute_multisig_transaction_success'),
+                type: 'success',
+            })
+        } catch (e: any) {
+            console.error('MultiSigTx::sign: Error', e)
+            dispatchNotification({
+                message: this.$t('notifications.execute_multisig_transaction_error'),
+                type: 'error',
+            })
+            throw e
+        }
+    }
+
+    async updateMultisigTxDetails() {
```

Comment:
After executing the transaction, displayed data are not properly update (i.e showing msig infos before the edit)

Reply:
Resolved